packages/wallets/src/features/accounts/screens/PersonalDetails/PersonalDetails.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { ReactNode } from 'react';
+import { useResidenceList } from '@deriv/api';
+import { WalletDropdown, WalletText, WalletTextField } from '../../../../components';
+import { accountOpeningReasonList } from './constants';
+import './PersonalDetails.scss';
+
+const PersonalDetails = () => {
+    const { data: residenceList } = useResidenceList();
+
+    return (
+        <div className='wallets-personal-details'>
+            <div className='wallets-personal-details__header'>
+                <WalletText align='center' as='h2' color='prominent' weight='bold'>
+                    Complete your personal details
+                </WalletText>
+                <WalletText align='center' size='xs'>
+                    Any information you provide is confidential and will be used for verification purposes only.
+                </WalletText>
+            </div>
+            <div className='wallets-personal-details__form'>
+                <WalletDropdown
+                    label='Citizenship*'
+                    list={residenceList.map(residence => ({
+                        text: residence.text as ReactNode,
+                        value: residence.value ?? '',
+                    }))}
+                    onSelect={() => null}
+                    type='comboBox'
+                    value={undefined}
```

Comment:
@farhan-nurzi-deriv in my opinion you don't need to pass undefined to value. Instead of this just change `value` type of `WalletDropdown` from this: `value: string | undefined;` to this: `value?: string;`

Reply:
@sergei-deriv the prop value for dropdown is required but the value itself can be undefined. Thats why. It just placeholder,later i will pass the actual value here